lib/Doctrine/ORM/Event/LifecycleEventArgs.php

```diff
@@ -15,23 +15,41 @@
  */
 class LifecycleEventArgs extends BaseLifecycleEventArgs
 {
+    /** @var object */
+    private $entity;
+
+    /** @var EntityManagerInterface */
+    private $entityManager;
+
+    public function __construct(object $entity, EntityManagerInterface $entityManager)
+    {
+        $this->entity        = $entity;
+        $this->entityManager = $entityManager;
+    }
+
+    public function getObject() : object
+    {
+        return $this->entity;
+    }
+
     /**
      * Retrieves associated Entity.
-     *
-     * @return object
      */
     public function getEntity()
```

Comment:
You removed return type annotation without adding return type. Same for following methods in this class